javascript/chubbyts/package.json

```
@@ -1,7 +1,7 @@
 {
   "dependencies": {
-    "@chubbyts/chubbyts-framework": "~1.3.0",
-    "@chubbyts/chubbyts-framework-router-path-to-regexp": "~1.0.1",
+    "@chubbyts/chubbyts-framework": "~1.4.0",
+    "@chubbyts/chubbyts-framework-router-path-to-regexp": "*",
     "@chubbyts/chubbyts-http": "~1.0.0",
```

is it ok to use something like this @dominikzogg ?
```suggestion
    "@chubbyts/chubbyts-http": "*",
```

---

potential risk of break if i would create a new major, but cause its not that big of an issue if it breaks (its only a benchmark) feel free to adapt.